roadmap

```diff
@@ -0,0 +1,9 @@
+# Roadmap
+## React version Block 2
+
+- Week 1: Problems-Diagnosis heading / Clinical notes heading / Spinner http://prntscr.com/h2qgfv
```

Comment:
thanks @kbeloborodko  what about swapping the Generic Plugin at the start , ie week 1?

Reply:
@tony-shannon we started on Tuesday and almost finished those headings. For Week 2 we also have all of the preparations in place, so idea was to replace the heading that we don't have internal specifications for at this point

Reply:
ok , so the generic plugin work will begin week of Nov 6th? 
I didnt understand that , the dates would help ..
that would be ok
thanks